templates/desktop/flavours.html

```diff
@@ -119,7 +119,7 @@ <h2 class="p-heading-icon__title">Kubuntu</h2>
     <div class="col-start-large-4 col-9">
       {{
         image (
```

Comment:
```suggestion
        image(
```

Reply:
@chillkang could you please remove the space in this function call and reformat the file? And please do the same for all similar places in this file. Thanks!